### PR TITLE
Adjust pom

### PR DESCRIPTION
**Pull request by original-codematrix**
Moved some versions and some compiling options to the `properties` to have one area to manage that kinda stuff.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -197,41 +244,16 @@
...
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
-                <configuration>
-                    <source>11</source>
-                    <target>11</target>
-                    <encoding>UTF-8</encoding>
-                    <compilerArgs>
-                        <arg>-parameters</arg>
```
Comment (MiniDigger) [inline pom.xml:223]: We need this compiler arg, you can't remove this

File: pom.xml [hunk 2]
```diff
@@ -6,14 +6,61 @@
...
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.testSource>${java.version}</maven.compiler.testSource>
+        <maven.compiler.testTarget>${java.version}</maven.compiler.testTarget>
+
+        <!-- encoding -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- dependencies -->
```
Comment (MiniDigger) [inline pom.xml:32]: Can you please sort out the webjars to their own section?

File: pom.xml (hunk 1 above)
Comment (original-codematrix) [reply pom.xml:223]: Re-added it.

File: pom.xml (hunk 2 above)
Comment (original-codematrix) [reply pom.xml:31]: Done. Also moved the bom version to an own area maybe some more boms will be used in the future.

File: pom.xml (hunk 1 above)
Comment (MiniDigger) [reply pom.xml:223]: that flag makes the compiler include method params names in the LVT, allowing jdbi, the DB mapper we use, to automagically map stuff to DB tables by looking at the names of the params

Comment (original-codematrix) [reply pom.xml:223]: Ah for that it is used :D

**Pull request merged**